Snakemake/workflow/rules/prepare_files_for_tsinfer.smk

```diff
@@ -1,115 +1,116 @@
+if config['ancestral_allele'] is not None:
+    ancestral_file = config['ancestral_allele']
+else:
+    ancestral_file = rules.combine_pos_ancestral.output	
+
+
+def getInputVcfFile_af(wildcards):
```

Comment:
Again, we are not using this function - I've just left it in as an exmaple - but shouold probably delete it